docs/release-notes.rst

```diff
@@ -283,6 +283,14 @@ The latest FAST plugin (v1.14) introduced a different format for defining depend
 
 ------
 
+**Duplicate Pool Members are displayed on FAST form when an existing application is updated**
+
+On update operation of FAST application, the FAST form displays duplicate pool member fields. These duplicate fields do not appear in the BIG-IP FAST Applications UI. This issue still exists when trying to reload the L4-L7 tab. If a user tries to submit the application again with those duplicate fields, ‘Pool members should not have duplicate items’ error is received from BIG-IP. 
```

Comment:
something like "When updating a FAST application..." Or "During the update operation of a FAST application, "